apps/src/blockly/addons/cdoXml.js

```diff
@@ -35,19 +35,47 @@ export default function initializeBlocklyXml(blocklyWrapper) {
   };
 
   blocklyWrapper.Xml.domToBlockSpace = function(blockSpace, xml) {
+    // To position the blocks, we first render them all to the Block Space
```

Comment:
Thanks for adding this! For my understanding, what is an "invisible block"?

Reply:
Basically it's what it sounds like. It's a block that is present (ie. the block XML shows it's present alongside other code in the workspace) but the student can't actually see it or interact with it.

Reply:
Oh got it, this is just general to rendering Blockly, not just instructions/hints